### PR TITLE
Try secrets files

### PR DESCRIPTION
**Pull request by evrardjp**

### CODE REVIEW

File: rpc-jobs/jobs.yaml [hunk 1]
```diff
@@ -670,10 +670,10 @@
         - text:
             credential-id: RPC_REPO_KEY
             variable: REPO_KEY
-        - text:
+        - secret-file:
```
Comment (hughsaunders) [inline rpc-jobs/jobs.yaml]: should be -file rather than secret-file

File: rpc-jobs/jobs.yaml [hunk 2]
```diff
@@ -670,10 +670,10 @@
...
             credential-id: REPO_GPG_SECRET_FILE
             variable: REPO_GPG_PRIVATE
-        - text:
+        - Secret file:
```
Comment (hughsaunders) [inline rpc-jobs/jobs.yaml]: should be -file rather than secret-file

**Comment by hughsaunders** [comment]
example here is comprehensive http://docs.openstack.org/infra/jenkins-job-builder/wrappers.html#wrappers.credentials-binding

**Comment by evrardjp** [comment]
Done.

**Pull request merged**